handler/area_types.go

```diff
@@ -183,13 +198,30 @@ func (h *AreaTypes) GetParents(w http.ResponseWriter, r *http.Request) {
 	resp.Count = res.Count
 	resp.TotalCount = res.TotalCount
 	for _, e := range res.Dataset.Variables.Edges[0].Node.IsSourceOf.Edges {
+
+		i := 0
+		if len(e.Node.Meta.ONSVariable.GeographyHierarchyOrder) > 0 {
+			i, err = strconv.Atoi(e.Node.Meta.ONSVariable.GeographyHierarchyOrder)
+			if err != nil {
+				h.respond.Error(ctx, w, http.StatusBadRequest, &Error{
```

Comment:
Probably not the best status code to return

Reply:
Status code amended to be internal server error.